### PR TITLE
Responsive website

### PR DESCRIPTION
**Pull request by ahadb802**
**My Portfolio Website For Responsive Desktop**
- Added Styles to the responsive website
- Added some Images to the images folder
- Added some new tags in index file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -249,7 +267,7 @@ <h1 class="about-title">About <br> Myself</h1>
                 </li> 
             </ul>
             <Button class="button-7">Get my resume</BUtton>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle:  Please make sure you use lowercase for HTML tag elements 
 - Kindly use `<div>` not `<DIV>` nor `<Div>`

File: stylesheet.css [hunk 2]
```diff
@@ -1,6 +1,7 @@
 * {
   box-sizing: border-box;
   background: #e5e5e5;
+  margin: 0;
 }
 
 button:hover {
```
Comment (bienvenuushindi) [inline stylesheet.css]: :red_circle: You have done well implementing the hover effect :clap: :100: . Kindly make sure your hover state reflects the Figma Design  as shown in the screenshot below :arrow_down: 

buuton

File: index.html [hunk 3]
```diff
@@ -135,9 +151,10 @@ <h3 class="year">2015</h3>
         
         </div>
     </article>
-    <article>
+    <article class="cards">
         <div class="container-2">
-            <a href="#" class="project-pics"><img class="project-pics" src="./images/Snapshoot Portfolio (2).png" alt=""></a>
+            <a href="#" class="mobile-pic" ><img class="project-pics" src="./images/Snapshoot Portfolio (3).png" alt=""></a>
+            <a href="#" class="desktop-pic" ><img class="project-pics" src="./images/3d.png" alt=""></a>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: Your project card images for the desktop version are not displaying. Please make sure the images are present and the paths are correct 

Your Design
noimage

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @ahadb802  ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔
- Nice Design. ✔


## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 4]
```diff
@@ -14,18 +14,31 @@
...
+        <a href="#">A.Ahad</a>
+        </div>
         <nav >
+            <ul>
+                <li><a href="#">Portfolio</a>
+                </li>
+                <li><a href="#">About</a></li>
+                <li>
+                    <a href="#">Contact</a>
+                </li>
```
Comment (Felix45) [inline index.html:29]: [ Optional ] Please consider making these links scroll to the respective sections of the page, `Portfolio` should scroll to the portfolio section `About` should scroll to the about section, `Contact` should scroll to the contact section. You can achieve this using the code below.
```HTML
<ul>
   <li><a href="#projects">Portfolio</a> </li>
     <li><a href="#about">About</a></li>
       <li><a href="#contact">Contact</a></li>
</ul>

<section id="portfolio"></section>
<section id="about"></section>
<section id="contact"></section>
```

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @ahadb802 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [ ] All linter checks are passing :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 
- [ ] Good descriptive commit messages :white_check_mark: 
- [ ] Nice UI design :clap: :clap: 

Congratulations-Memes-FEATURED

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**